packages/notification-center/src/i18n/languages/fr.ts

```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const FR: ITranslationEntry = {
+  translations: {
+    notifications: 'Avis',
+    markAllAsRead: 'tout marquer comme lu',
```

Comment:
Honestly this is just a small detail
```suggestion
    markAllAsRead: 'Tout marquer comme lu',
```